lib/data_update_scripts/20200709154442_update_broadcastable_for_broadcasts.rb

```diff
@@ -0,0 +1,20 @@
+module DataUpdateScripts
+  class UpdateBroadcastableForBroadcasts
+    def run
+      Broadcast.find_each do |cast|
+        case cast.broadcastable_type
+        when "Welcome"
+          broadcastable = WelcomeNotification.create
+          result = cast.update(broadcastable: broadcastable)
+          next if result
+
+          Honeybadger.context(broadcast_id: cast.id, errors: cast.errors_as_sentence)
+          Honeybadger.nofity("Broadcast Update Failed")
```

Comment:
```suggestion
          Honeybadger.notify("Broadcast Update Failed")
```

This won't actually notify otherwise 😛

Reply:
Great catch!